nestjs/src/courses/course-schedule/course-schedule.service.ts

```diff
@@ -360,7 +360,7 @@ export class CourseScheduleService {
   private getEventStatus(courseEvent: CourseEvent) {
     const startTime = (courseEvent.dateTime as Date).getTime();
     const endTime = Number(courseEvent.endTime) ?? startTime + (courseEvent.duration ?? 60) * 1000 * 60;
```

Comment:
Btw I am not sure this is correct. After conversion to number left part of this expression will never be `null` or `undefined`, so right part will not work, I think proper version looks like this, could you please fix it?
```suggestion
    const endTime = Number(courseEvent.endTime) || startTime + (courseEvent.duration ?? 60) * 1000 * 60;
```